### DISCUSSION TITLE
RC xApp not working with DL traffic

### DISCUSSION THREAD

**Discussion by pengnist** [srsran/oran-sc-ric#22] [General]
Hi Community,

I am trying to use the simple_rc_xapp.py and encountered some issues. My setup includes the dockerized oran-sc-ric, open5gs core, srsran gnb and srsran_4G ue:

1. When generating DL traffic from 5G core to UE using iperf3, launching the RC xapp does not change the DL throughput. The data rate stayed the same as configured at iperf3 server. My understanding was that the PRB ratio should have alternated between 50% and 5%, thus the DL data rate should have alternated between high and low values every 5 seconds. 
2. When generating the UL traffic from UE to core using iperf3, I observed the correct UL traffic as configured, and the DL data rate was around 600kb/s. However, after the xApp was launched, the DL rate started alternating between high and low rates (in unit of MB/s, with high rate close to what I set for UL). The UL rate was correct. This observation was on the dockerized Grafana at gNB, and also at the srsran 4G UE statistics. However, when launching the KPM xApp, it shows the DL rate stayed at around 600kb/s.

I am wondering if any troubleshooting can be performed on my side and where I should start.

Thanks

**Comment by pgawlowicz** [comment]
check the answer here: https://github.com/srsran/oran-sc-ric/issues/21
